README.md

```diff
@@ -2,7 +2,20 @@
 
 [![Build Status](https://travis-ci.org/ITISFoundation/osparc-simcore.svg?branch=master)](https://travis-ci.org/ITISFoundation/osparc-simcore)
 
-## Deploy
+## Overview
+
+[![service-web](docs/img/service-web.svg)](http://interactive.blockdiag.com/?compression=deflate&src=eJxdjs0KwjAQhO99imXPFtNbpcYXkR7ys2hw6UqSKiK-uymxiF7nm28Yy-IuPpgTPBsAvJPdOg40ZYRjOpsr6UkyjUOB_tEmirfgKH0YaEDHMnsshX993x5qsEgUyx4bS6x3qu824IQlastz3f4pFtGHSC5LXCXsleqw3ljRUvteGprXG1PtQR0)
+
+
+## Deploy in-place
+
+```bash
+  make build

```

Comment:
I don't think this is correct as make build, builds the docker-compose.yml and up uses the docker-compose.yml + docker-compose.deploy.yml files

Very nice graphs!